### PR TITLE
GEOMESA-2903 Log HBase zookeeper connection

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
from what I can tell, there's no good way to see what resources are loaded through a configuration object. the resources that it contains may or may not point to actual files, it only determines that when it tries to parse them. they are also relative names (i.e. `hbase-site.xml`), the full path is only known internally when they are parsed. so the only other sane option i see is to print out the entire conf, but that seems very verbose.

**Comment by elahrvivaz** [comment]
kerberos info is already logged

File: geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseConnectionPool.scala [hunk 1]
```diff
@@ -101,6 +101,7 @@ object HBaseConnectionPool extends LazyLogging {
       ProvidedConnection(ConnectionParam.lookup(params))
     } else {
       val conf = getConfiguration(params)
+      logger.debug(s"Connecting to HBase instance at ${conf.get(HConstants.ZOOKEEPER_QUORUM)}")
```
Comment (jnh5y) [inline geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseConnectionPool.scala:104]: I wonder if we should add logic to add warning if the ZooKeeper is localhost.  

Meh, probably too much.

Comment (elahrvivaz) [reply geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseConnectionPool.scala:104]: we do have a warning about that already actually.

**Pull request merged**